extension.js

```diff
@@ -32,7 +32,7 @@ if (!global.workspace_manager) {
  */
 var modules = [
     'tiling', 'navigator', 'keybindings', 'scratch', 'liveAltTab', 'utils',
-    'stackoverlay', 'app', 'kludges', 'topbar', 'settings','gestures'
+    'stackoverlay', 'app', 'kludges', 'topbar', 'settings','gestures', 'animations'
```

Comment:
I would also recommend splitting the changes into two separate commits, it would improve experience working with the history: merging, rebasing, reverting changes and so forth.

https://www.aleksandrhovhannisyan.com/blog/atomic-git-commits/